### ISSUE TITLE
Accessibility: text elements should have reasonable contrast ratio

### ISSUE THREAD

**Issue by ameliejyc** [AbdullahChaudhry/week-5-CADJ#32]
5 contrast ratio improvements:

<h3 id=​"h5o-2">​Randomise your dreams!​</h3>​
<label for=​"genre">​Select a Genre:​</label>​
<label for=​"year">​Select a Year:​</label>​
<section title=​"search-results" id=​"releasedate">​Release date: 1990-08-03​</section>​
<section title=​"search-results" id=​"overview">​…​</section>​
